frontend/src/app/modules/common/autocomplete/create-autocompleter.component.ts

```diff
@@ -33,6 +33,7 @@ import {I18nService} from "core-app/modules/common/i18n/i18n.service";
 import {CurrentProjectService} from "core-components/projects/current-project.service";
 import {PathHelperService} from "core-app/modules/common/path-helper/path-helper.service";
 import {HalResource} from "core-app/modules/hal/resources/hal-resource";
+import {AddTagFn} from "@ng-select/ng-select/lib/ng-select.component";
 
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/opf/openproject/blob/877156b0e9a0ec9b6103ddc6d1696b94294cfe7e/frontend/src/app/modules/common/autocomplete/create-autocompleter.component.ts#L37"></span>Please use <code>ChangeDetectionStrategy.OnPush</code> for this component</td>
    </tr>
  </tbody>
</table>